src/components/ButtonAuth.jsx

```diff
@@ -2,19 +2,21 @@ import Image from "next/image";
 import { Button } from "react-flatifycss";
 
 import "flatifycss/dist/css/flatify-min.css";
-import styles from "./ButtonAuth.module.css";
 
-const ButtonAuth = (props) => {
-    const { icon, alt, width, height } = props;
+const ButtonAuth = ({ icon, alt, width, height, text }) => {
+    const styles = {
+        backgroundColor: "white",
+        border: "2px solid black",
+        borderRadius: "0.5rem",
+        boxShadow: "2px 2px #1a1a1a",
+    };
```

Comment:
To better use tailwind, I think this should be written as utility classes. for the boxShadow you can use the arbitrary feature which will look something like this: shadow-[2px 2px #1a1a1a].

Reply:
I've tried it before but it did not give the same view of the figma design